cms/djangoapps/contentstore/utils.py

```diff
@@ -296,7 +296,8 @@ def get_group_display_name(user_partitions, xblock_display_name):
     """
     for user_partition in user_partitions:
         for group in user_partition['groups']:
-            if str(group['id']) in xblock_display_name:
+            # Temporary hack until TNL-6731 is addressed.
```

Comment:
Note to reviewers: this issue will be fixed before this PR merges. FYI @mushtaqak